java/vector/src/main/java/org/apache/arrow/vector/ipc/WriteChannel.java

```diff
@@ -41,6 +41,10 @@
 public class WriteChannel implements AutoCloseable {
   private static final Logger LOGGER = LoggerFactory.getLogger(WriteChannel.class);
 
+  private static byte[] zeroBytes = new byte[8];
```

Comment:
This could be marked `final`

Reply:
Revised. Thank you for the good suggestion.